peer.go

```diff
@@ -438,6 +441,9 @@ func (p *peer) loadActiveChannels(chans []*channeldb.OpenChannel) error {
 				time.NewTicker(time.Minute)),
 			BatchSize:    10,
 			UnsafeReplay: cfg.UnsafeReplay,
+			UpdateFeeInterval: func() time.Duration {
+				return time.Duration(prand.Intn(60)+11) * time.Minute
```

Comment:
Magic numbers should be lifted up into constants.

Reply:
Fixed.